integration/sync_test.go

```diff
@@ -226,8 +227,9 @@ func (s *SyncSuite) TestYamlUntagged(c *check.C) {
 	defer os.RemoveAll(tmpDir)
 	dir1 := path.Join(tmpDir, "dir1")
 
-	image := "alpine"
-	imageRef, err := docker.ParseReference(fmt.Sprintf("//%s", image))
+	registry := "k8s.gcr.io"
+	image := "pause"
+	imageRef, err := docker.ParseReference(fmt.Sprintf("//%s/%s", registry, image))
```

Comment:
An unrelated and absolutely non-blocking note: The second use of `imageRef` should probably refer to `v2DockerRegistryURL+"/"+imagePath`, otherwise the `localTags` checks are meaningless.

Reply:
Filed #1150 , mostly as a reminder to myself to fix that.